### PR TITLE
fix invalid concated shape in dot merge pass.

### PR DESCRIPTION
**Pull request by qiuxiafei**

### CODE REVIEW

File: scripts/python/tao_build.py [hunk 1]
```diff
@@ -342,6 +342,8 @@ def _write(line, cmd="build"):
         _write("--host_jvm_args=-Djdk.http.auth.tunneling.disabledSchemes=", cmd = "startup")
         logger.info("configuring tao_bridge with bazel ......")
 
+    with cwd(root):
+        execute('rm -f tao_compiler/decoupling/version.h')
```
Comment (Orion34-lanbo) [inline scripts/python/tao_build.py:346]: Why deleting version.h here?

Comment (wyzero) [reply scripts/python/tao_build.py:346]: +1

Comment (qiuxiafei) [reply scripts/python/tao_build.py:346]: if `version.h` exists (i.e. re-configure project), it will fail at L355, so delete it in advance.

Comment (qiuxiafei) [reply scripts/python/tao_build.py]: anyway, I removed this explicit `rm -f` and use force copy on L354.

**Review by wyzero — APPROVED** [review]
LGTM

**Pull request merged**